src/clj_kondo/impl/analysis.clj

```diff
@@ -97,3 +97,10 @@
                        :filename filename
                        :lang (when (= :cljc (:base-lang ctx)) (:lang ctx))
                        :id (:id binding)))))
+
+(defn reg-keyword-usage! [ctx filename usage]
+  (when-let [analysis (:analysis ctx)]
```

Comment:
Should we also check for `:analyze-keywords?` here?

Reply:
done